src/cider/exec/nextgen/context/CodegenContext.h

```diff
@@ -87,6 +88,7 @@ using AggExprsInfoVector = std::vector<AggExprsInfo>;
 
 struct CodegenOptions {
   bool needs_error_check = false;
+  jitlib::CompilationOptions co = jitlib::CompilationOptions{};
 };
```

Comment:
need to move `needs_error_check` into compilationOptions?

Reply:
@bigPYJ1151 any suggestion?

Reply:
I remember there is different definition between `CodegenOptions ` and `CompilationOptions` per @bigPYJ1151 ?